server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -317,6 +318,9 @@ protected Node(final Environment initialEnvironment,
             Settings tmpSettings = Settings.builder().put(initialEnvironment.settings())
                 .put(Client.CLIENT_TYPE_SETTING_S.getKey(), CLIENT_TYPE).build();
 
+            // Enabling shard indexing backpressure node-attribute
+            tmpSettings = addShardIndexingBackPressureAttributeSettings(tmpSettings);
```

Comment:
Why not to do this on one shot? Fe

```
Settings tmpSettings = Settings.builder().put(initialEnvironment.settings())
           .put(Client.CLIENT_TYPE_SETTING_S.getKey(), CLIENT_TYPE)
           .put(NODE_ATTRIBUTES.getKey() + SHARD_INDEXING_PRESSURE_ENABLED_ATTRIBUTE_KEY, "true")
.build();
```

Reply:
Just for abstraction/readability, and not to make the to constructor too bulky. I am fine either ways if you suggest so.

Reply:
:+1: Using builder style looks more concise to be fair, at least - the API was designed this way